_episodes/10-zonal-statistics.md

```diff
@@ -19,76 +19,41 @@ keypoints:
 
 ~~~
 import rioxarray
-ndvi = rioxarray.open_rasterio("NDVI.tif")
-ndvi_sq = ndvi.squeeze()
-~~~
-
-Let's also read the crop fields vector data from our saved `cropped_field.shp` file and view the CRS information.
-
-~~~
-field = gpd.read_file('cropped_field.shp')
-field.crs
+ndvi = rioxarray.open_rasterio("NDVI.tif").squeeze()
 ~~~
 {: .language-python}
 
-~~~
-<Derived Projected CRS: EPSG:28992>
-Name: Amersfoort / RD New
-Axis Info [cartesian]:
-- X[east]: Easting (metre)
-- Y[north]: Northing (metre)
-Area of Use:
-- name: Netherlands - onshore, including Waddenzee, Dutch Wadden Islands and 12-mile offshore coastal zone.
-- bounds: (3.2, 50.75, 7.22, 53.7)
-Coordinate Operation:
-- name: RD New
-- method: Oblique Stereographic
-Datum: Amersfoort
-- Ellipsoid: Bessel 1841
-- Prime Meridian: Greenwich
-~~~
-{: .output}
+Let's also read the crop fields vector data from our saved `fields_cropped.shp` file.
 
-In order to use the vector data as a classifier for our raster, we need to convert the vector data to the appropriate CRS. We can perform the CRS conversion from the vector CRS (EPSG:28992) to our raster `ndvi` CRS (EPSG:32631) and view the data with:
 ~~~
-field_to_raster_crs = field.to_crs(ndvi.rio.crs)
-field_to_raster_crs
+import geopandas as gpd
+fields = gpd.read_file('fields_cropped.shp')
 ~~~
 {: .language-python}
 
+In order to use the vector data as a classifier for our raster, we need to convert the vector data to the appropriate CRS. We can perform the CRS conversion from the vector CRS (EPSG:28992) to our raster `ndvi` CRS (EPSG:32631) with:
 ~~~
-category	gewas	gewascode	jaar	status	geometry
-0	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((634234.009 5807461.338, 634232.049 5...
-1	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((634514.198 5807699.177, 634504.207 5...
-2	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((633115.463 5808493.238, 633109.078 5...
-3	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((634803.514 5808081.449, 634809.802 5...
-4	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((634184.289 5807370.958, 634200.036 5...
-...	...	...	...	...	...	...
-4867	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((631384.726 5809352.385, 631383.343 5...
-4868	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((635240.367 5806904.896, 635245.819 5...
-4869	Grasland	Grasland, blijvend	265	2020	Definitief	POLYGON ((636074.093 5816782.787, 636123.922 5...
-4870	Grasland	Grasland, tijdelijk	266	2020	Definitief	POLYGON ((627526.751 5816828.877, 627674.251 5...
-4871	Grasland	Grasland, natuurlijk. Hoofdfunctie landbouw.	331	2020	Definitief	POLYGON ((642317.485 5813024.516, 642326.058 5...
-4872 rows × 6 columns
+# Uniform CRS
+fields = fields.to_crs(ndvi.rio.crs)
```

Comment:
Maybe use a different variable name and avoid overwriting? I think overwriting can create confusion to participants: if they skip one line of code, they can be lost